### ISSUE TITLE
Propagate moderation actions

### ISSUE THREAD

**Issue by Lacerkage** [Lacerkage/PastaBot#5]
If a thread gets deleted or edited on the imageboard, the respective status published by the bot should reflect those changes.

**Comment by loynet** [comment]
From my current understanding, events (log entries) have to be fetched on a board basis. This is potentially problematic since the bot would have to fetch the logs from all boards that have been the source of, at least, one _status_.
Furthermore, the log related endpoints ([1](http://fatchan.gitgud.site/jschan-docs/#board-log-list) and [2](http://fatchan.gitgud.site/jschan-docs/#board-log-day)) bring additional challenges. The first only provides the number of entries per day and the second, which returns a somewhat detailed list of events, does it by date (e.g. GET https://ptchan.org/i/logs/09-27-2022.json returns a list of /i/ events that happened yesterday).

TLDR: To know what has to be propagated, one must fetch logs from the current day in every board that has been the source of a _status_* and this has to be performed at least once per day (which is not ideal for obvious reasons, we should try to propagate changes every 30 minutes or so) in order safely ignore previous days changes.
*Threads are ephemeral in imageboards and it’s probably okay to assume that if a thread has not been deleted after _t_ time, there’s nothing wrong with it. Maybe we can watch only “fresh” _status_.

... I don’t see how one could implement a such solution without persistent memory of threads (their id, board, and probably more) that have been posted.
